api/openapi.yaml

```diff
@@ -873,6 +873,521 @@ paths:
                         example: No path could be found
         400:
           description: The request body is invalid
+
+  /projects/:
+    post:
+      tags:
+        - projects
+      summary: Create a new project
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ProjectRequest"
+      responses:
+        201:
+          description: The created project
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProjectResult"
+    get:
+      tags:
+        - projects
+      summary: Paginated list of projects
+      parameters:
+        - in: query
+          name: oredering
+          schema:
+            type: string
+            enum:
+              [
+                "name",
+                "-name",
+                "creation_date",
+                "-creation_date",
+                "last_modification",
+                "-last_modification",
+              ]
+        - in: query
+          name: name
+          schema:
+            type: string
+          description: Filter projects by name
+        - in: query
+          name: description
+          schema:
+            type: string
+          description: Filter projects by description
+        - in: query
+          name: tags
+          schema:
+            type: string
+          description: Filter projects by tags
+        - in: query
+          name: page
+          schema:
+            type: integer
+            minimum: 0
+          description: Page number
+        - in: query
+          name: page_size
+          schema:
+            type: integer
+            maximum: 10000
+            minimum: 1
+            default: 100
+          description: Number of elements by page
+      responses:
+        200:
+          description: the project list
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    type: number
+                  next: {}
+                  previous: {}
+                  results:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        schema:
+                          $ref: "#/components/schemas/ProjectResult"
+
+  /projects/{project_id}/:
```

Comment:
I get an error when I try to render the file on editor.swagger.io:

> Declared path parameter "project_id" needs to be defined as a path parameter at either the path or operation level

Reply:
I still get the error, it's not fixed. I don't know if it's a problem on the editor side, does this render properly on another editor?

Reply:
Yes, this render properly in vscode swagger editor extension :(

Reply:
I'll commit a fix then

Reply:
I tried on editor.swagger.io and I just fixed it. Can you check please ? :)

Reply:
I've pushed a fix, it seems that the problem was that path parameters weren't properly specified. As a bonus, they can be moved to the root of the endpoint

Reply:
Oh we fixed it twice :/